lua/lazyvim/plugins/extras/formatting/prettier.lua

```diff
@@ -58,7 +58,7 @@ return {
             end
             if enabled[ctx.filename] == nil then
               enabled[ctx.filename] = vim.fs.find(function(name, path)
-                return name:match("^%.prettierrc%.") or name:match("^prettier%.config%.")
+                return name:match("^%.prettierrc%.*.*") or name:match("^prettier%.config%.*.*")
```

Comment:
right!
but probably only the first pattern has to change. Just add a `?` and drop your other changes

Reply:
Sorry was outside and didn't see this until now.